doc/nrf/known_issues.rst

```diff
@@ -935,6 +935,10 @@ KRKNWK-15846: Android CHIP Tool crashes when subscribing in the :guilabel:`LIGHT
   Also, the Subscription window in the :guilabel:`LIGHT ON/OFF & LEVEL CLUSTER` contains faulty GUI layout (overlapping captions) used when passing minimum and maximum subscription interval values.
   This affects the Android CHIP Tool revision used for the |NCS| v2.2.0, v2.1.1, and v2.1.2 releases.
 
+KRKNWK-15913: "user" field in the factory data set is not properly parsed. That field should have type of MAP instad of BSTR.
+
+  **Workaround:** Manually cherry-pick and apply commit with fix to ``sdk-connectedhomeip`` (commit hash: ``TODO add SHA``).
```

Comment:
```suggestion

.. rst-class:: v2-2-0 v2-1-2 v2-1-1

KRKNWK-15913: Factory data set parsing issues
  The ``user`` field in the factory data set is not properly parsed. The field should be of the ``MAP`` type instead of the ``BSTR`` type.

  **Workaround:** Manually cherry-pick and apply commit with fix to ``sdk-connectedhomeip`` (commit hash: ``TODO add SHA``).
```